recipe/meta.yaml

```
@@ -35,10 +36,10 @@ test:
     # for tests
     - iso8601 >=1,<2
     - mock >=4,<5
-    - pytest >=6,<7
+    - pytest >=6,<8
```

https://github.com/graphql-python/graphene/blob/82903263080b3b7f22c2ad84319584d7a3b1a1f6/setup.py#L48
```suggestion
    - pytest >=8,<9
```

---

Thanks for the review! I actually used these exact versions  in my initial commit. However,  The newer test dependency versions aren't available in the defaults channel. The version you reviewed with relaxed dependencies was a workaround I had to implement because the CI tests were failing.  With these relaxed versions, all builds and tests pass successfully.

I've now re-applied your suggestions to match upstream exactly, but as expected, the CI tests failed.

Shall we
  1. Add conda-forge channel for test dependencies?
  2. Revert to the previous commit?
  
  Which approach would you prefer? The test dependencies only affect package testing during the build process, not the actual package functionality for end users.

---

```suggestion
    # locking to lower test dependencies while we don't satisfy all pytest versions
    - pytest >=6,<8
```

---

Thank you